Sources/App/WebView/WebViewController.swift

```diff
@@ -24,6 +24,10 @@ class WebViewController: UIViewController, WKNavigationDelegate, WKUIDelegate, U
 
     var keepAliveTimer: Timer?
     private var initialURL: URL?
+    private let biometricOverlay: UIView = {
```

Comment:
let's move this to either the WebViewWindowController or something on the Scene level. if it's not related to the function of the WebView, it should go elsewhere